### ISSUE TITLE
Error in v1.36.0

### ISSUE THREAD

**Issue by VadVergasov** [platomav/MCExtractor#26]
Python 3.6.8
Windows 10 x64 Pro 1903

```
Error:
Traceback (most recent call last):
  File "MCE.py", line 67, in <module>
    newline=sys.stdout.newlines, line_buffering=sys.stdout.line_buffering, write_through=sys.stdout.write_through)
  File "C:\Program Files\Python36\lib\site-packages\colorama\ansitowin32.py", line 29, in __getattr__
    return getattr(self.__wrapped, name)
AttributeError: '_io.TextIOWrapper' object has no attribute 'write_through'
```

Tried v1.35.0 all runs successfully.

**Comment by platomav** [comment]
Thank you for the report.

The "write_through" error seems to be some weird Python < 3.7 omission. It was supposedly added at 3.3 but not really. However, even when removed, Python < 3.7 behaves in a weird way when it comes to console redirection. The ANSI codes for color are shown and not removed. These are all specific issues at Python under Windows when using Unicode. Finally, at Python 3.7, a simple new & robust function was added which vastly improves that Windows Unicode weirdness.

Starting from the next version of MCE, the script will require Python 3.7 as a minimum.

**Comment by platomav** [comment]
The issue has now been resolved at commit 264ffd2
